choose/choose.go

```diff
@@ -61,6 +61,9 @@ func (c choose) Update(msg tea.Msg) (m tea.Model, cmd tea.Cmd) {
 		)
 
 	case tea.KeyMsg:
+		if play.ValidateBindings(msg) {
```

Comment:
There is no problem on that screen. Let's revert it

Reply:
But on this screen the user can press k/j/q/. And if the layout is incorrect, the user will not even be able to exit the application.